examples/quantizing_moe/qwen_moe_w4a16.py

```diff
@@ -0,0 +1,122 @@
+import torch
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from llmcompressor.transformers import oneshot
+from llmcompressor.transformers.compression.helpers import calculate_offload_device_map
+from llmcompressor.modifiers.quantization import GPTQModifier
+
+# select a Mixture of Experts model for quantization
+MODEL_ID = "Qwen/Qwen1.5-MoE-A2.7B-Chat"
+
+# adjust based off number of desired GPUs
+# if not enough memory is available, some layers will automatically be offloaded to cpu
+device_map = calculate_offload_device_map(
+    MODEL_ID,
+    reserve_for_hessians=True,
+    num_gpus=2,
```

Comment:
This should use the number of gpus available, not a hardcoded number

Reply:
All of the other uses of calculate_offload_device_map seems to use hardcoded values - what should we do here?

Reply:
Actually I think it's fine, if 2 is the suggested number that we'd want users to have available.